src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle.bib

```diff
@@ -1,5 +1,6 @@
 @article{,
       address = {Cent States Forest Expt Stn, Columbus, OH USA},
+      alt-title = {Ecology},
```

Comment:
improvement: if `alt-title` matches `journal`, then remove `alt-title`. Maybe, a cleanup at the end.

Reply:
Should the same apply if ```booktitle``` matches ```alt-title```?

Reply:
Yes, I have done that.